AI-and-Analytics/Getting-Started-Samples/IntelModin_GettingStarted/IntelModin_GettingStarted.ipynb

```diff
@@ -47,6 +47,60 @@
     "import time"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ****** Do not change the code in this cell! It verifies that the notebook is being run correctly! ******\n",
+    "\n",
+    "def verify_and_print_times(pandas_time, modin_time):\n",
+    "    if modin_time < pandas_time:\n",
+    "        print(f\"Modin was {pandas_time / modin_time:.2f}X faster than stock Pandas!\")\n",
+    "        return\n",
+    "    print(\n",
+    "        f\"Oops, stock Pandas appears to be {modin_time / pandas_time:.2f}X faster than Modin in this case. \"\n",
+    "        \"This is unlikely but could happen sometimes on certain machines/environments/datasets. \"\n",
+    "        \"One of the most probable reasons is the excessive amount of partitions being assigned to a single worker. \"\n",
+    "        \"You may visit Modin's optimization guide in order to learn more about such cases and how to fix them: \"\n",
+    "        \"\\nhttps://modin.readthedocs.io/en/latest/usage_guide/optimization_notes/index.html\\n\\n\"\n",
+    "        \"But first, verify that you're using the latest Modin version, also, try to use different executions, \"\n",
+    "        \"for basic usage we recommend non-experimental 'PandasOnRay'\\n\"\n",
+    "        \"Current execution is:\"\n",
+    "    )\n",
+    "    try:\n",
+    "        from modin.utils import get_current_execution\n",
+    "\n",
+    "        execution = get_current_execution()\n",
```

Comment:
[The comment according `get_current_execution`](https://github.com/oneapi-src/oneAPI-samples/pull/843/files#r800373473) was applied